test/test_cache.py

```diff
@@ -135,6 +151,7 @@ def test_combined_profile_update_check_has_changed(self):
         uuid = 'FAKEUUID'
         uep = Mock()
         uep.has_capability = Mock(return_value=True)
+        print(self.current_profile)
```

Comment:
Do we need this line? Or is it forgotten debug output?

Reply:
Good point.